src/components/widgets/SelectWidget.js

```diff
@@ -32,6 +36,11 @@ function SelectWidget({
   onChange
 }) {
   const {enumOptions} = options;
+
+  const emptyOption = !multiple && (!schema.default || schema.default.length === 0)
```

Comment:
@lablancas That option should work to also. In this case, I followed the updates `react-jsonschema-form` has pushed since we forked which was to check if its a multiple select and if the default exists.